edkahara/iReporter
# Ft create v2 admin change status endpoint #162423641

__What does this PR do__
Create an endpoint that allows only an admin to change the status of a report.

__Description of tasks to be completed.__
Create an api endpoint that facilitates the changing of the status of a report by an admin. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-admin-change-status-endpoint-#162423641 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set up environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetests.

__Relevant pivotal tracker stories.__
#162423641
